neqo-transport/src/frame.rs

```diff
@@ -660,8 +670,11 @@ mod tests {
 
     fn just_dec(f: &Frame, s: &str) {
         let encoded = Encoder::from_hex(s);
-        let decoded = Frame::decode(&mut encoded.as_decoder()).unwrap();
-        assert_eq!(*f, decoded);
+        if let Ok(decoded) = Frame::decode(&mut encoded.as_decoder()) {
+            assert_eq!(*f, decoded);
+        } else {
+            panic!("Failed to decode frame");
+        }
```

Comment:
```suggestion
        let decoded = Frame::decode(&mut encoded.as_decoder()).expect("Failed to decode frame");
        assert_eq!(*f, decoded);
```

If you need a specific message, this is neater.